build/docker/toybox-config

```diff
@@ -21,7 +21,7 @@ CONFIG_TOYBOX_GETRANDOM=y
 #
 # CONFIG_BASENAME is not set
 # CONFIG_CAL is not set
-# CONFIG_CAT is not set
```

Comment:
looks like these are manually modified changes. Did you follow the `make oldconfig` and `make menuconfig` path?

Reply:
I chnaged with the result generated from menuconfig with the old config, but idk why some are disabled..
> CONFIG_TOYBOX_CONTAINER=y
CONFIG_TOYBOX_FIFREEZE=y

> # CONFIG_TOYBOX_CONTAINER is not set
# CONFIG_TOYBOX_FIFREEZE is not set

Reply:
that's ok